uber/templates/preregistration/preregbase.html

```diff
@@ -18,7 +15,7 @@
     };
     setInterval(checkCap, 1000 * 1 * 90);
 
-{% if COLLECT_FULL_ADDRESS %}
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
Haha, true.  Though actually I was planning on making a ``magprime`` repo this year since there probably will be some event-specific functionality.  This would replace the ``magprime_emails`` repo, since at the very least it'll contain custom MAGFest emails.